doc/source/whatsnew/v0.25.0.rst

```diff
@@ -241,6 +241,7 @@ Reshaping
 - Bug in :func:`pandas.merge` adds a string of ``None`` if ``None`` is assigned in suffixes instead of remain the column name as-is (:issue:`24782`).
 - Bug in :func:`merge` when merging by index name would sometimes result in an incorrectly numbered index (:issue:`24212`)
 - :func:`to_records` now accepts dtypes to its `column_dtypes` parameter (:issue:`24895`)
+- Bug in :func:`concat` where order of ``OrderedDict`` is not respected (:issue:`21510`)
```

Comment:
Based just on the release note, I have two questions:

1. What argument is being passed as an OrderedDict here?
2. Does this change apply to regular dicts in Python 3.6+?

Reply:
1. Do you mean which argument in the concat function is being passed as an OrderedDict?
2. Yes, because I make use of a function that also checks for dicts that are in Python 3.6+ and they follow the same order as the OrderedDict.

Reply:
1. Yes. Presumably `objs`, but a reader may potentially think `keys`.
2. It'd be good to mention that in a parenthetical.

Reply:
Bug in :func:`concat` where order of ``OrderedDict`` (and ``dict`` in Python 3.6+) is not respected, when passed in as  ``objs`` argument (:issue:`21510`)

Would you suggest changing the release note to this?

Reply:
yes